docs/rules/no-japanese-character.md

```diff
@@ -22,14 +22,19 @@ var str = this.getIntlMessage("i18n.key"); // using an i18n tool instead of regu
 
 ## Rule Options
 
-The rule takes one option, an object, which has one key `includeIdentifier` having boolean values `true` or `false`.
+The rule takes one option, an object, which has two keys `includeIdentifier`, `callee.excludes`. `includeIdentifier` has boolean values `true` or `false`, and `callee.excludes` has string array.

```

Comment:
Open discussion:

I know the `callee` is coming from AST, and it's totally fine with me. But do you think using `function` or `method` would be easier for users to understand? Would something like this be more intuitive:
```
{
    "includeIdentifier": true,
    "excludeArgumentsFrom": ["some", "function", "names"]
  }
```

What do you think?

Reply:
Thank you for your reply.
As you pointed out, `callee` is a term familiar to AST, but it might be a bit confusing for users. 

Based on the option name you proposed, I'm considering the name `excludeArgsForFunctions`. 
What do you think? 
I thought this naming might more explicitly indicate the specification of function names.

Reply:
Thank you!
Then I will modify the option name to `excludeArgsForFunctions`.